cms/core/migrations/0006_update_user_groups.py

```diff
@@ -0,0 +1,192 @@
+from django.conf import settings
+from django.db import migrations
+from wagtail.models import PAGE_PERMISSION_CODENAMES, GroupCollectionPermission, GroupPagePermission
```

Comment:
please move this to the op functions as 

`GroupPagePermission = apps.get_model("wagtailcore.GroupPagePermission")` and `GroupCollectionPermission = apps.get_model("wagtailcore.GroupCollectionPermission")`